libs/pxweb2-ui/src/lib/components/Radio/Radio.module.scss

```diff
@@ -0,0 +1,62 @@
+@use '../../../../style-dictionary/dist/scss/fixed-variables.scss' as fixed;
+@use '../../text-styles.scss';
+
+.radioGroup {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  align-self: stretch;  
+}
+
+.container {
+  align-items: flex-start;
+  display: flex;
+  width:  100%;
+  padding: 10px fixed.$spacing-2;
+  position: relative;
+  padding-left: 44px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  font-family: PxWeb-font-400;
+  letter-spacing: 0em;
+  color: var(--px-color-text-default);
+  line-height: 24px;
+  font-size: 16px;
```

Comment:
When we set values for CSS for fonts, we should use REM and not PX. Both line 24 and 25 here.

With REM the users can decide what font sizes they want in the settings of the browser, so if they set "Large" font-size, all sizes increase on any website. PX does not adjust itself in that way.